app/src/components/integrations/elements/integrationSettings/connectionSection/connectionSection.jsx

```diff
@@ -131,46 +154,101 @@ export class ConnectionSection extends Component {
     } = this.props;
-    const isConnectionFailed = !!failedConnectionMessage;
+
+    const availableProjectIntegrations = projectIntegrations[pluginName] || [];
 
     return (
-      <div className={cx('connection-section')}>
-        <h3 className={cx('block-header')}>{formatMessage(messages.connectionTitle)}</h3>
-        <div className={cx('connection-status-block', { 'connection-failed': isConnectionFailed })}>
-          <span className={cx('connection-status')}>
-            <span className={cx('status-icon')}>
-              {Parser(isConnectionFailed ? InfoIcon : CheckIcon)}
-            </span>
-            {formatMessage(
-              isConnectionFailed ? messages.connectionFailedMessage : messages.connectedMessage,
-            )}
-          </span>
-          {isConnectionFailed && (
-            <span className={cx('failed-connection-message')}>{failedConnectionMessage}</span>
-          )}
-        </div>
-        {editAuthConfig && (
-          <Fragment>
-            <div className={cx('connection-status-block', 'auth-info-block')}>
-              {editAuthConfig.content}
+      <>
+        {!connected && (
+          <div className={cx({ 'with-global-message': blocked })}>
+            <SystemMessage
+              header={formatMessage(messages.connectionFailedHeader)}
+              mode="error"
+              caption={formatMessage(messages.connectionFailedCapture)}
+            >
+              {formatMessage(messages.connectionFailedDescription, { pluginName })}
+            </SystemMessage>
+          </div>
+        )}
+        {availableProjectIntegrations.length > 0 && blocked && (
+          <SystemMessage header={formatMessage(messages.warningMessage)} mode="warning">
+            {formatMessage(messages.warningMessageDescription)}
+          </SystemMessage>
+        )}
+
+        <div
+          className={cx('connection-section', {
+            'connection-section-with-message':
+              !connected || (availableProjectIntegrations.length > 0 && blocked),
+          })}
+        >
+          <div className={cx('integration-info-block')}>
+            <div className={cx('sub-header-block')}>
+              <div className={cx('general-info')}>
+                <h1>{name}</h1>
+                <p>
+                  {creator} on {moment(creationDate).format('ll')}
+                </p>
+              </div>
+              <div
+                className={cx('connection-block', {
+                  'connection-block-failed': !connected,
+                })}
+              >
+                {!connected ? (
+                  <>
+                    {Parser(ErrorIcon)} <p>{formatMessage(messages.connectionFailedMessage)}</p>
+                  </>
+                ) : (
+                  <>
+                    {Parser(Tick)} <p>{formatMessage(messages.connectedMessage)}</p>
+                  </>
+                )}
+              </div>
```

Comment:
Please try to short it
`{Parser(connected ? Tick : ErrorIcon)} <p>{formatMessage(connected ? messages.connectedMessage : messages.connectionFailedMessage)}</p>`
Also in general if you process both(true\false) cases better start with true to avoid unnecessary negation(!)